neuralnetwork/machinelearning/Train.java

```diff
@@ -2,51 +2,73 @@
 
 import neuralnetwork.dataset.Data;
 import neuralnetwork.dataset.Dataset;
+import neuralnetwork.initializemethod.HeInitialization;
 import neuralnetwork.initializemethod.InitializeMethod;
 
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
 import java.util.Calendar;
 
 public class Train {
+
     private NeuralNetwork neuralNetwork;
-    private Dataset trainDataset = null;
-    private Dataset testDataset = null;
-    private int batch;
-    private int epoch;
+    private Dataset trainDataset;
+    private Dataset testDataset;
+    private int batch = 4; // default option
```

Comment:
batch's deafult value should be 1 to express SGD, and epoch should be 1,  backup false